docs/website/docs/guides/deployment-configurations/index.md

```diff
@@ -37,13 +37,40 @@ When compiling programs, a list of target backends must be specified via
 | `cuda`         | NVIDIA GPU support via PTX for CUDA | `cuda` |
```

Comment:
For L36, `vulkan` / `vulkan-spirv`, given both are recognized; also to be consistent with the following list.